packages/README.md

```diff
@@ -0,0 +1,73 @@
+# Jetpack packages for Development
+
+These Composer packages offer a unified codebase that we will share among projects under the Jetpack brand.
+
+## Installing Composer
```

Comment:
I don't know if it's worth mentioning this here since Composer is now a general requirement for Jetpack development. 

Maybe we should add this here instead?
https://github.com/Automattic/jetpack/blob/master/docs/development-environment-cli-support.md
https://github.com/Automattic/jetpack/blob/master/docs/development-environment.md#L14
https://github.com/Automattic/jetpack/blob/master/docs/development-environment.md#L41

Reply:
Good point. I'll move the installation instructions to `development-environment.md` and then I'll add here a reference to the corresponding paragraph there.

Reply:
Done.